app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/item/VaultItemScreen.kt

```diff
@@ -163,7 +163,7 @@ fun VaultItemScreen(
                     { viewModel.trySendAction(VaultItemAction.Common.CloseClick) }
                 },
                 actions = {
-                    if (state.isCipherDeleted) {
+                    if (state.isCipherDeleted && state.canDelete) {
```

Comment:
So you can only restore a cipher if it can be deleted?

Reply:
Maybe it should be `canManage` to be more generic?

Reply:
Ah. Nope. This check shouldn't be here.